codebasin/file_source.py

```diff
@@ -513,22 +511,28 @@ def c_file_source(fp, relaxed=False, directives_only=False):
         yield curr_line
 
     total_sloc += curr_line.physical_reset()
-    if not relaxed and not cleaner.state == ["TOPLEVEL"]:
+
+    # Even if code is technically wrong, we should only fail when necessary.
+    parsing_failed = not cleaner.state == ["TOPLEVEL"]
+    if continued:
+        log.warning("backslash-newline at end of file")
+        parsing_failed = False
+
+    if parsing_failed:
         raise RuntimeError(
-            "Parser must end at top level without 'relaxed' mode.",
+            "Parsing failed. Please open a bug report at: "
+            "https://github.com/intel/code-base-investigator/issues/new?template=bug_report.yml.",  # noqa: E501
```

Comment:
Remove the period from the the end of the link, otherwise it resolves to a blank issue template as opposed to throwing a 404!

Reply:
Good catch!  Fixed in 8d541523c50cf0812922b4cc455e3a49f16751c6.